stripe/src/main/java/com/stripe/android/PaymentConfiguration.kt

```diff
@@ -0,0 +1,79 @@
+package com.stripe.android
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class PaymentConfiguration private constructor(val publishableKey: String) : Parcelable {
```

Comment:
i don't think Parcelize is ok w/ private constructors?
https://github.com/Kotlin/KEEP/blob/master/proposals/extensions/android-parcelable.md

Reply:
good call, fixed!